src/compiler/statement/CatchClause.ts

```diff
@@ -0,0 +1,33 @@
+import * as ts from "typescript";
+import * as errors from "./../../errors";
+import {Block} from "./Block";
+import {Statement} from "./Statement";
+import {Node} from "../common";
+import {VariableDeclaration} from "./VariableDeclaration";
+
+export const CatchClauseBase = Node;
+export class CatchClause extends CatchClauseBase<ts.CatchClause> {
+
+    /**
+     * Gets this catch clause's block.
+     */
+    getBlock() {
+        return this.getNodeFromCompilerNode(this.compilerNode.block) as Block;
+    }
+
+    /**
+     * Gets this catch clause's variable declaration or undefined if none exists.
+     */
+    getVariableDeclaration() {
```

Comment:
Interesting. There's a remove method on VariableDeclaration that will throw in this scenario. I'll fix that.